pkg/client/unversioned/request.go

```diff
@@ -164,6 +169,12 @@ func (r *Request) SubResource(subresources ...string) *Request {
 		r.err = fmt.Errorf("subresource already set to %q, cannot change to %q", r.resource, subresource)
 		return r
 	}
+	for _, s := range subresources {
```

Comment:
see examples called out in https://github.com/kubernetes/kubernetes/pull/16717#issuecomment-153275275